cmd/run/message_input.go

```diff
@@ -100,17 +106,6 @@ func (mi *MessageInput) sendAction() {
 	mi.reset()
 }
 
-func (mi *MessageInput) pasteAction() {
```

Comment:
Copy and paste is supported by `TextArea` by default using the `(*tview.TextArea).SetClipboard` function for customization.